package.json

```diff
@@ -34,7 +36,18 @@
     "stylelint-config-sass-guidelines": "^5.2.0"
   },
   "scripts": {
-    "postinstall": "grunt gessoSetup",
-    "component": "node component.js"
+    "component": "node component.js",
+    "postinstall": "grunt gessoSetup"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,json,md,prettierrc,scss,yml}": [
+      "node ./node_modules/.bin/prettier --write",
```

Comment:
I don't think `node ./node_modules/.bin/` is required here -- `lint-staged` adds locally-installed Node binaries to $PATH when it executes: https://github.com/okonet/lint-staged#what-commands-are-supported

Reply:
That’s what I had originally, but it didn’t work. On one of the Github issues I found someone from the Prettier team recommend using `
node ./node_modules/.bin/prettier` instead.